dcl-common/src/scene.rs

```diff
@@ -0,0 +1,77 @@
+use serde::{Deserialize, Serialize};
+use super::Parcel;
+
+#[derive(Debug, Deserialize, Serialize, Default, Clone)]
+pub struct Scene {
+   pub name: String,
+   pub entities: Vec<Entity>,
+   pub parcels: Vec<Parcel>,
+   pub path: Option<String>,
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone)]
+pub struct Entity {
+    pub name: String,
+    pub components: Vec<Component>,
+}
+
+#[derive(Deserialize, Serialize, Debug, Clone)]
+pub enum Component{
+    Transform(Transform),
```

Comment:
I'm wondering if using enums like this will be problematic if someone wants to implement this in other languages. Let's discuss.